slack_pull_reminder.py

```diff
@@ -29,9 +35,36 @@
 """
 
 
+def get_teams(organization, team_names=None):
+    if not team_names:
```

Comment:
why is this fallback necessary? wouldn't `team_names` being `None` imply that we don't want any filtering whatsoever?